samples/app-auth/nodejs/README.md

```diff
@@ -184,44 +187,6 @@ Notes:
 5. Click **Save**.
 6. Set the environment variable `AZUREAD_CONNECTIONNAME` to the name that you chose for this OAuth connection.
 
-### [Optional] Using LinkedIn
-
-1. Follow the instructions [here](https://docs.microsoft.com/en-us/linkedin/shared/authentication/authorization-code-flow?context=linkedin/context#step-1-configure-your-application) to create and configure a LinkedIn application for OAuth 2.
-2. In "Authorized Redirect URLs", add `https://token.botframework.com/.auth/web/redirect`.
-3. Note your app's "Client ID" and "Client Secret".
-4. Navigate to your Azure bot's service page on the [Azure Portal](https://ms.portal.azure.com/#blade/HubsExtension/BrowseResourceBlade/resourceType/Microsoft.BotService%2FbotServices).
-5. Click **Settings**.
-6. Under **OAuth Connection Settings** near the bottom of the page, click **Add Setting**.
-7. Fill in the form as follows:
-
-    1. For **Name**, enter a name for your connection (e.g., "LinkedIn")
-    2. For **Service Provider**, select **LinkedIn**.
-    3. For **Client id**, enter the client ID for your LinkedIn app.
-    4. For **Client secret**, enter the client secret for your LinkedIn app.
-    5. For **Scopes**, enter `r_liteprofile r_emailaddress`.
-
-8. Click **Save**.
-9. Set the environment variable `LINKEDIN_CONNECTIONNAME` to the name that you chose for this OAuth connection.
-
-### [Optional] Using Google
-
-1. Obtain OAuth2 client credentials from the [Google API Console](https://console.developers.google.com). Enable access to the [Google People API](https://developers.google.com/people/).
-2. In "Authorized redirect URLs", add `https://token.botframework.com/.auth/web/redirect`.
-3. Note your app's "Client ID" and "Client Secret".
-4. Navigate to your Azure bot's service page on the [Azure Portal](https://ms.portal.azure.com/#blade/HubsExtension/BrowseResourceBlade/resourceType/Microsoft.BotService%2FbotServices).
-5. Click **Settings**.
-6. Under **OAuth Connection Settings** near the bottom of the page, click **Add Setting**.
-7. Fill in the form as follows:
-
-    1. For **Name**, enter a name for your connection (e.g., "Google")
-    2. For **Service Provider**, select **Google**.
```

Comment:
You removed linked in related auth, so please make sure that images/gif are also updated.

Reply:
updated images and gif